routes/frameSkip.js

```diff
@@ -0,0 +1,38 @@
+const FrameSkipResult = require('../models/frameSkipResult');
+
+module.exports = function(app) {
+    app.post('/frameSkipResult', (req, res) => {
+        const { buildNum, buildUrl, frameSkipPercentage } = req.body;
+
+        console.log(`received frame skip pct ${frameSkipPercentage}`
+            + ` for build num ${buildNum}`
+            + ` url ${buildUrl}`);
+
+        FrameSkipResult.create({
+            buildNum,
+            buildUrl,
+            frameSkipPercentage
+        })
+        .then(createdFrameSkipResult => { // eslint-disable-line no-unused-vars
+            res.sendStatus(200);
+        })
+        .catch(err => {
+            console.log(`Error inserting frameSkipResult: ${err}`);
+            res.sendStatus(500);
+        });
+    });
+
+    app.get('/frameSkipResults', (req, res) => {
+        FrameSkipResult.findAll().then(frameSkipResults => {
+            console.log('got results: \n',
+               frameSkipResults.map(result => result.toJSON()));
+            res.send(frameSkipResults.map(result => result.toJSON()));
```

Comment:
Normally I'm not gonna complain much about duplication in a greenfield repo but I do feel this map() result should be stored in a variable because it's doing a couple actions on the data.

Reply:
i think i'll just get rid of the log